core-developers/memorialization.rst

```diff
@@ -0,0 +1,134 @@
+.. _memorialize-core-developer:
+
+===============
+Memorialization
+===============
+
+Rationale
+=========
+
+When a core developer passes away, memorializing accounts helps create
+a space for remembering the contributor and protects against attempted
+logins and fraudulent activity.
+
+The process
+===========
+
+The memorialization process is performed by a member of the PSF staff
+with administrative access to current and historical systems where
+core developers have access.
+
+After the status of the core developer in question is confirmed,
+access to the systems listed below is revoked and some changes are
+made to how the user displays to others.
+
+To respect the choices that someone made while alive, we aim to preserve
+content of their accounts without changes after they've passed away.
+To support the bereaved, in some instances, we may remove or change
+certain content when the legacy contact or family members request it.
+
+GitHub
+------
+
+* The user is removed from the `python/ <https://github.com/orgs/python/>`_
```

Comment:
I'm not sure how it works, but I wouldn't be surprised if removing someone from an organization doesn't remove them from a GitHub Environment or GitHub Security Advisory where they are referenced by username (as opposed to being a part of a team).

Reply:
So far I checked and the person that this process was documented for was in fact removed from all places where they were referred to by name as well, since the only way to be referenced was to either be a team member or an "outside collaborator" on the organization. I'll leave it at that for now, if we find the next time that we need refinement, we'll look into that.